coordinator/src/notifications.rs

```diff
@@ -120,6 +122,10 @@ fn build_notification(kind: &NotificationKind) -> fcm::Notification<'_> {
             notification_builder.title("Your position has expired");
             notification_builder.body("Close your position.");
         }
+        NotificationKind::PositionLiquidated => {
+            notification_builder.title("Your position has been liquidated");
+            notification_builder.body("Close your position.");
```

Comment:
We need the user to open the app to collaboratively execute the liquidation.

> Your position has been liquidated 📉😞 

Don't know if you'd have to be sad about it. It could also be that the counterparty got liquidated and you made a huge profit :)

> Open the app to try again!

I copied it from the expired position notification as its the same action the user needs to do, which is "open the app". I guess we should change it to this message :)

Reply:
https://github.com/get10101/10101/pull/2374

Reply:
> Don't know if you'd have to be sad about it. It could also be that the counterparty got liquidated and you made a huge profit :)

I guess the message should depend on who got liquidated!